graalpy-install-package/src/main/resources/META-INF/native-image/native-image.properties

```diff
@@ -1 +1 @@
-Args = -H:-CopyLanguageResources 
```

Comment:
This change is not right, the language resources are bundled into the native image, they shouldn't be copied

Reply:
Somehow the agent isn't working correctly in the CI run - it doesn't generate the resource-config.json file that would be needed to build the native image correctly.

Reply:
Is there a way of forcing the resource-config.json file? It's throwing the following exception

>Traceback (most recent call last):
  File "pyfigletwrapper", line 1, in <module>
ModuleNotFoundError: No module named 'pyfigletwrapper'
Exception in thread "main" org.graalvm.polyglot.PolyglotException
	at org.graalvm.polyglot/org.graalvm.polyglot.Context.eval(Context.java:402)
	at com.oracle.example.graalpy.Main.createPyfigletProxy(Main.java:141)
	at com.oracle.example.graalpy.Main.main(Main.java:57)

Is that a resources problem?

Reply:
When you run the app, you don't seem to use the `native` profile. I think you need it to active the tracing agent in your current setup (`mvn -Pnative exec:exec`).

Reply:
It's at line 46 of the YAML file
`mvn exec:exec -Pnative &`

Reply:
Right, I assume you tried this locally and it works? Maybe killing the process keeps the agent from writing out the config files. Maybe, you could let the app terminate gracefully after some seconds? You could even make that configurable via some Java property or environment variable.

Reply:
Found the ticket: https://github.com/graalvm/native-build-tools/issues/584